autoload/go/jobcontrol.vim

```diff
@@ -93,7 +93,7 @@ endfunction
 " it'll be closed.
 function! s:on_exit(job_id, data)
   let std_combined = self.stderr + self.stdout
-  if empty(std_combined)
+  if jobwait([a:job_id])[0] > 0
```

Comment:
Ok, valid arguments. Right now we only use it for `build` and `test`. If there is anything wrong with any upstream app, we should fix it instead of.